docs/mapview.md

```diff
@@ -64,6 +64,8 @@ To access event data, you will need to use `e.nativeEvent`. For example, `onPres
 |---|---|---|
 | `animateToRegion` | `region: Region`, `duration: Number` |
 | `animateToCoordinate` | `coordinate: LatLng`, `duration: Number` |
+| `animateToBearing` | `bearing: Number`, `duration: Number` |
+| `animateToViewingAngle` | `angle: Number`, `duration: Number` |
```

Comment:
Add note about it only being available on Google maps for iOS.

Reply:
I added support for the IOS native map...Basically, I do (for both animations) the difference is that in one I set some property on the camera before: Pitch (ViewAngle on GoogleMaps) and Heading (Bearing on GoogleMaps):

```
[AIRMap animateWithDuration:duration/1000 animations:^{
    [mapView setCamera:mapCamera animated:YES];
}];
```

Reply:
@christopherdro I addressed your PR comments :)